### ISSUE TITLE
Setup Farcaster app for CI/Heroku

### ISSUE THREAD

**Issue by rbennettcw** [hicommonwealth/commonwealth#9117]
After #9101 is complete, the Farcaster code will be added back, as a new separate app. In order to deploy, some additional work is required.

## Acceptance Criteria
- Farcaster app should build in CI
- Farcaster app should deploy to its own dyno w/ subdomain setup

**Comment by rbennettcw** [comment]
This is no longer relevant since farcaster will be part of the main app again.
